tools/coqdoc/cpretty.mll

```diff
@@ -951,20 +967,21 @@ and escaped_html = parse
         { backtrack lexbuf }
   | _   { Output.html_char (lexeme_char lexbuf 0); escaped_html lexbuf }
 
-and verbatim depth inline = parse
-  | nl ">>" space* nl { Output.verbatim_char inline '\n'; Output.stop_verbatim inline }
-  | nl ">>" { Output.verbatim_char inline '\n'; Output.stop_verbatim inline }
-  | ">>" { Output.stop_verbatim inline }
-  | "(*" { Output.verbatim_char inline '(';
-           Output.verbatim_char inline '*';
-           verbatim (depth+1) inline lexbuf }
-  | "*)" { if (depth == 0)
-           then (Output.stop_verbatim inline; backtrack lexbuf)
-           else (Output.verbatim_char inline '*';
-                 Output.verbatim_char inline ')';
-                 verbatim (depth-1) inline lexbuf) }
-  | eof { Output.stop_verbatim inline }
-  | _ { Output.verbatim_char inline (lexeme_char lexbuf 0); verbatim depth inline lexbuf }
+and verbatim_block = parse
+  | nl ">>" space* nl { new_lines 2 lexbuf; Output.verbatim_char false '\n'; Output.stop_verbatim false }
+  | nl ">>" { new_lines 1 lexbuf; Output.verbatim_char false '\n'; Output.stop_verbatim false }
```

Comment:
Add a warning that this really should be followed by a newline.

The two calls to `verbatim_block` are followed by `doc_bol` or `doc_list_bol`.

Reply:
Good remarks!

Reply:
I added a warning. Thanks a lot for the very relevant comments!